core/src/messages/agent.rs

```diff
@@ -375,6 +375,8 @@ pub struct ResourceLimits {
     /// Total cpu time allocated to container    
     #[serde_as(as = "Option<DurationSeconds>")]
     pub cpu_time_limit: Option<Duration>,
+
+    pub memory_limit: Option<i64>,
```

Comment:
What’s the unit here? How about something like `memory_limit_mb` or similar depending on what the unit is?

Reply:
changed to memory_limit_bytes with comment